intTests/test_llvm_bound_check/test.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+set -e
+
+# We want to ensure there isn't a memory store failure and this is caught during type checking of Setup Value
+if $SAW bound.saw | grep -q "Memory store failed"; then
```

Comment:
I'd be more inclined to check for the presence of this error message instead, i.e.,

```suggestion
if $SAW bound.saw | grep "array type index out of bounds"; then
```

That way, if we change the error message on SAW's end, we're much more likely to notice it here (and update the test accordingly). If we changed the `Memory store failed` message in Crucible, on the other hand, this test wouldn't notify us.